superset/daos/dashboard.py

```diff
@@ -195,11 +195,6 @@ def update_charts_owners(model: Dashboard, commit: bool = True) -> Dashboard:
     @classmethod
     def delete(cls, items: Dashboard | list[Dashboard], commit: bool = True) -> None:
         item_ids = [item.id for item in get_iterable(items)]
-        # bulk delete, first delete related data
-        for item in get_iterable(items):
-            item.slices = []
-            item.embedded = []
```

Comment:
This doesn't seem necessary per the way the [relationship is defined](https://github.com/apache/superset/blob/master/superset/models/dashboard.py#L166-L170).